### ISSUE TITLE
Clean up Runner Tests using AutoFixture instead of Reflection

### ISSUE THREAD

**Issue by jzabroski** [fluentmigrator/fluentmigrator#1071]
Based on discussion in PR https://github.com/fluentmigrator/fluentmigrator/pull/1069#discussion_r330639521

**Comment by BigYellowHammer** [comment]
Started working on that one.
Is this what you want? (I just changed first 3 unit tests)
https://github.com/BigYellowHammer/fluentmigrator/commit/6727398a0b2abed44bf6cfce9f57740ff2679617

**Comment by jzabroski** [comment]
Yes, except we may be able to eventually move the AutoFixture pieces into a
TestBase class. Don't worry about that, let's first clean up one test. Baby
steps.

Thanks for taking care of this.
